frontend/packages/component-library/src/carousel/stories/Carousel.story.tsx

```diff
@@ -46,11 +46,7 @@ const MultipleTemplate: StoryFn<{components: CarouselProps[]}> = args => (
         style={{maxWidth: '800px', margin: '0 auto', marginBlock: '2rem'}}
         key={index}
       >
-        <Carousel
-          {...componentArg}
-          key={componentArg.carouselId}
-          children={componentArg.children}
```

Comment:
This is a leftover from the early days of building this component and is unneeded.